Makefile

```diff
@@ -2039,7 +2050,7 @@ benchmark_apps: $(BENCHMARK_APPS)
 	@for APP in $(BENCHMARK_APPS); do \
 		echo ;\
 		echo Benchmarking $${APP} for ${HL_TARGET}... ; \
-		make -C $(ROOT_DIR)/apps/$${APP} \
+		make -C $(ROOT_DIR)/apps/$${APP} test \
```

Comment:
Why did you add this to the `benchmark_apps` target? This one was fine before.

Reply:
Accidental. Removed.